### ISSUE TITLE
Complete actions following accessibility testing workshops

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#2661]
## What
Using the Task List component as an example, run two workshops on Zoom with the community on how to accessibility test a component.

## Why
The community have expressed an interest in learning how we do manual accessibility testing of a component.

## Who needs to work on this
David, Imran

## Who needs to review this
N/A

## Done when
- [x] Questions from workshops collected
- [ ] Move documentation that's stored in Padlet to a more secure long-term storage format
- [x] (Optional) Demo portion of the workshop re-recorded when David's voice is feeling better
- [x] Transcripts completed
- [x] Recording of workshop shared

**Comment by davidc-gds** [comment]
The final set of retro notes (for the 'focus not obscured' workshop) cannot be found. We believe they were also done via Padlet but can't locate the board.

As such, this ticket is now complete, with the 2 'archive' Google Doc files above ✅
